OracleWebLogic/dockerfiles/12.2.1.1/Dockerfile.developer

```diff
@@ -63,8 +62,8 @@ RUN chmod a+xr /u01 && \
     chown oracle:oracle -R /u01 && \
     rm /u01/$FMW_JAR /u01/$FMW_PKG /u01/oraInst.loc /u01/install.file
 
-USER oracle
-WORKDIR $ORACLE_HOME 
+COPY container-scripts/* /u01/oracle/

```

Comment:
Best practice is to copy each script instead of an entire directory. This is so the Docker build process doesn't invalidate the cache every time it builds.

See the **Cacheability** section in the official Docker library repo for more info on this: https://github.com/docker-library/official-images/blob/master/README.md